Documentation imp 2 (#49)
znicholls/silicone: README.rst

.. sec-begin-long-description
.. sec-begin-index

> Silicone is a Python package which can be used to infer predictions about certain types

Reviewer: ```suggestion
Silicone is a Python package which can be used to infer
```

> of emissions from other predicted emissions. It is intended to ensure more integrated

Reviewer: ```suggestion
emissions from other emissions data. It is intended to 'infill' integrated
```

> assessment models (IAMs) have predictions of all climate-relevant emissions.

Reviewer: ```suggestion
assessment model (IAM) data so that their scenarios quantify more climate-relevant emissions than is natively reported by the IAMs themselves.
```

> It does this by comparing the incomplete emissions set to completed data.

Reviewer: ```suggestion
It does this by comparing the incomplete emissions set to complete data. It uses the relationships within the complete data to make informed infilling estimates of otherwise missing emissions timeseries.
```

Yuck suggestion from me but hopefully inspires something better

For example, it can add emissions of aerosol precurors based on carbon dioxide emissions

> and infilling nitrous oxide emissions based on methane, or splitting HFC emissions

Reviewer: ```suggestion
and infill nitrous oxide emissions based on methane, or split HFC emissions
```

Installation
------------

> This Python package can be installed directly from github. The release version is hosted at

Reviewer: Let's skype about this. Best practice is to release like is done for [scmdata](https://scmdata.readthedocs.io/en/latest/development.html#releasing).

That means users can then install simply by doing `pip install silicone` or `conda install silicone`. If they want a specific version, they can specify e.g. `pip install silicone==0.2.1`. With releases on pypi (pip) and conda, we don't have to worry about a 'release version' and a 'development version'. I would suggest we simply do away with the znicholls/silicone (when you're ready) and move it all to GranthamImperial.

Author: This seems like a good idea, but to be done in a separate pull request. Currently what's written there is true, so I'm inclined to stick with that and update it later.
